docs/content.zh/docs/ops/metrics.md

```diff
@@ -1428,6 +1428,16 @@ Note that the metrics are only available via reporters.
       <td>Maximum recorded duration of a single consecutive period of the task being in the hard back pressure state in the last sampling period. Please check softBackPressuredTimeMsPerSecond and hardBackPressuredTimeMsPerSecond for more information.</td>
       <td>Gauge</td>
     </tr>
+    <tr>
+      <td>mailboxThroughputPerSecond</td>
```

Comment:
`mailsPerSecond` would be more consistent with existing metrics? `taskThreadActionsPerSecond`?

Reply:
I would keep the prefix with `mailbox`. Do you think `mailboxMailsPerSecond` is okay?